pkgs/by-name/id/idmail/package.nix

```diff
@@ -38,14 +40,17 @@ rustPlatform.buildRustPackage rec {
   RUSTC_BOOTSTRAP = 1;
   RUSTFLAGS = "--cfg=web_sys_unstable_apis";
 
+  OPENSSL_CONF = lib.optionalString stdenv.isDarwin "${openssl.out}/etc/ssl/openssl.cnf";
```

Comment:
Please split these changes to its own commit, and add comment on why is this necessary.